auth/src/main/java/com/hms/lib/commonmobileservices/auth/google/GoogleAuthServiceImpl.kt

```diff
@@ -263,8 +309,18 @@ class GoogleAuthServiceImpl : AuthService {
         TODO("Not yet implemented")
     }
 
-    override fun getPhoneCode(var1: String?, var2: String?): Work<Unit> {
```

Comment:
please give the parameter names more clearly.

Reply:
resolved.